tools/packaging/scripts/lib.sh

```diff
@@ -181,6 +181,11 @@ sha256sum_from_files() {
 	fi
 }
 
+calc_shim_v2_files_sha256sum() {
+	local files="${repo_root_dir}/src/runtime"
+	sha256sum_from_files "$files"
+}
+
```

Comment:
Gaby, small change of plans.

Maybe it's better to drop this one, and instead use `$(get_last_modification "${repo_root_dir}/runtime")`.